.env

```diff
@@ -38,4 +38,4 @@ NEXT_PUBLIC_ETYPE_MECHANISM_MAP_ID=https://bbp.epfl.ch/data/bbp/mmb-point-neuron
 NEXT_PUBLIC_CONN_INITIAL_RULES_FILE_ID=https://bbp.epfl.ch/neurosciencegraph/data/7e2f41a0-3d84-4142-a38e-734117acb33d
 NEXT_PUBLIC_CONN_INITIAL_PARAMS_FILE_ID=https://bbp.epfl.ch/neurosciencegraph/data/2924ab22-086e-42d5-96ee-f9ce46ccf4b4
 
-NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_51P6uAFFE4Bi50cLlatJIc0fUPsP0jQkaCCJ8TTkIYOOLIrLzxX1M9p1kVD11drNqsF9p7yiaumWJ8UHb3ptJJRXB00y3qjYReV
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY="pk_test_51QjjHBKGUR5u3ofLgNUOpljnvy27UTTpkhwgsLiwK9xlNjnR7CZfiMjtZWMjgN7GW3eDyzMJ7Z1pIqC9LiwkfQRX00ebb5c9XI"
```

Comment:
Probably it's best to move this to `.env.development`, so there is no change this can end up in the production build.